HW10/HW10_Interfaces_Abstract_classes/src/Main.java

```diff
@@ -0,0 +1,67 @@
+import java.util.Objects;
+import java.util.HashMap;
+public class Main {
+    public static void main(String[] args) {
+
+       // Створіть кілька користувачів.
+        // Додайте контакти для кожного користувача.
+        // Відправте повідомлення між користувачами.
+        // Виведіть список повідомлень кожного користувача із зазначенням статусу.
+        // Викличте метод для "прочитування повідомлень" від імені користувачів,
+        // які отримали повідомлення та перевірте статус цих повідомлень після прочитання.
+
+        User tania = new User("Tania");
+        User artem = new User("Artem");
+        User mum = new User("Mum");
+
+        tania.addContact("0999999999", artem);
+        tania.addContact("099994444", mum); // Tania - 2 contacts
+        artem.addContact("099999554", tania);
+
+
+        Message firstMessage = new Message("Hello, it's me");
+        Message secondMessage = new Message("Bye Bye");
+        Message thirdMessage = new Message("Spam this is spam");
+        Message fourthMessage = new Message("enemy");
+        Message fifthMessage = new Message("test");
+        Message six = new Message("new text");
+
+        tania.sendMessage(firstMessage, artem);
+        artem.getMessage(firstMessage);
```

Comment:
Ideally some potential method `.receiveMessage(Message message)` would handle this situation as usually we don't `receive` messages intentionally - they're being sent to us (it's a passive behavior of a user which can be triggered by somebody when they send the message)

On the other hand, users are reading messages by themselves (this is an active behavior), so it can stay as a direct call from the main method. What if I just want to read the latest message? Ideally I'd just call .readLastMessage() method from the user and it would retrieve the message for me and change its status to "READ" so it can't be read twice